components/tools/OmeroWeb/omeroweb/settings.py

```diff
@@ -508,12 +508,10 @@ def leave_none_unset_int(s):
          "Enable and disable the OMERO.web public user functionality."],
     "omero.web.public.url_filter":
         ["PUBLIC_URL_FILTER",
-         r'^/(?!webadmin)',
+         r'a^',
          re.compile,
-         ("Set a URL filter for which the OMERO.web public user is allowed to"
-          " navigate. The idea is that you can create the public pages"
-          " yourself (see OMERO.web framework since we do not provide public"
-          " pages.")],
+         ("Set a filter for URLS which the public user is allowed to access."
```

Comment:
Is it worth stating that this is a regular expression? e.g. "Set a regular expression for URLs". Filter could be some custom syntax, whereas calling it a regular expression tells the user the expected syntax.

Reply:
Perhaps a comment, such as: `r'(?#This regular expression matches nothing)a^',`

Reply:
Thanks. Updated both.